src/dev/code_coverage/docs/team_assignment/README.md

```diff
@@ -29,15 +29,15 @@ The last 3 lines above ONLY attribute files in those paths for purpose of code c
 We create a data file containing all paths in the repo, with a team assigned.   
 Example Team Assignments Block: 
 ```
-x-pack/plugins/security_solution/common/constants.ts siem
-x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.test.ts siem
-x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.ts siem
+x-pack/solutions/security/solutions/security/plugins/security_solution/common/constants.ts siem
+x-pack/solutions/security/solutions/security/plugins/security_solution/common/detection_engine/build_exceptions_query.test.ts siem
+x-pack/solutions/security/solutions/security/plugins/security_solution/common/detection_engine/build_exceptions_query.ts siem
 ...
 ```
 
 ## Team Assignment Data File Usage (During Code Coverage Ingestion) 
 Subsequently, we use the data file during ingestion.
 We search the data file, for any given "coveredFilePath"
  - Given the above assignments block, and lets say the "coveredFilePath" during ingestion is 
-   - `x-pack/plugins/security_solution/common/constants.ts`
+   - `x-pack/solutions/security/solutions/security/plugins/security_solution/common/constants.ts`
```

Comment:
Again, duplicate path bits for `solutions/security`. Should probably be something like
```suggestion
   - `x-pack/solutions/security/plugins/security_solution/common/constants.ts`
```